### DISCUSSION TITLE
emoji won't show

### DISCUSSION THREAD

**Discussion by smjt2000** [starship/starship#3578] [Q&A] [answered]
Hi
I'm on ArtixLinux and started using starship a few minuets age.
emojis won't show in my terminal.
I tried to install some fonts like noto-sans-symbols noto-emoji but when I'm in a python directory the snake emoji don't show correctly.

**Comment by matchai** [comment, marked as answer]
I'd recommend taking a look at #1600, which has some guidance around ArchLinux configuration.

**Reply by smjt2000** [reply]
didn't work

**Reply by matchai** [reply]
What did you try? What didn't work?
What terminal emulator are you using?

**Reply by smjt2000** [reply]
I don't know how, but problems solved. I installed the alacritty font with **pacman** but emojis still not showing.
decide to install it from github, just cloned the repository and installed packages that required, logged out from system and next time I logged in I got emojis but I have an error from the font it my terminal.

I'm using konsole

**Comment by smjt2000** [comment]
@matchai I have a problem.
on qtile everything is good, but when I'm on Kde Plasma I have some issues with starship.
I'm using Konsole and the cursor is placed wrong. it's far from where characters place when I'm typing and when the cursor reach end of line, next chars will place at the next line.
so I uploaded some screenshots, just look at the cursor and where characters placed.

another issue is emojis won't show correctly in Plasma only. starship is working so fine on qtile.
[first](https://ibb.co/cht5WCN)
[second](https://ibb.co/M5MpWvk)
[third](https://ibb.co/h243qZN)

last thing I wanna say is I'm not good in English so please eli5

**Reply by matchai** [reply]
The cursor location is likely a result of Konsole mishandling the widths of emoji characters.
I'd recommend taking a look at this configuration someone shared: https://github.com/starship/starship/issues/2099#issuecomment-758613241

**Reply by smjt2000** [reply]
it's working but there is some issues that not very important.

Thank you
